src/vanilla/store2.ts

```diff
@@ -243,6 +243,7 @@ const flushPending = (pending: Pending) => {
 // for debugging purpose only
 type DevStoreRev4 = {
   dev4_get_internal_weak_map: () => WeakMap<AnyAtom, AtomState>
+  dev4_get_mounted_atoms: () => IterableIterator<AnyAtom>
```

Comment:
I think it's fine to return the Set, given that our contract is it's read-only.
```suggestion
  dev4_get_mounted_atoms: () => Set<AnyAtom>
```

Reply:
Makes sense! Updated it in [this commit](https://github.com/pmndrs/jotai/pull/2594/commits/b10f78126fa5a941aefa25284ea6d9950c46d7a2)